### ISSUE TITLE
Bug: numbered list within toggle block

### ISSUE THREAD

**Issue by ados8** [docmost/docmost#122]
When using numbered lists outside of toggle block there's no issue, works as expected.
When within a toggle block it cannot count above 9 (or into double digits).
![Numbered list bug](https://github.com/user-attachments/assets/ab480ebf-3333-4518-9550-3a41f4b809b7)

**Comment by olivierIllogika** [comment]
It seems the number are there, but partially hidden by css.

The change below seems to fix it and I haven't found any side effects yet.  I'm guessing a better fix would be to style the `li::marker` with `overflow-x: visible`, but I couldn't make that work.

```diff
--- a/apps/client/src/features/editor/styles/details.css
+++ b/apps/client/src/features/editor/styles/details.css
@@ -51,7 +51,6 @@
     [data-type="detailsContainer"] {
       flex: 1;
       margin-left: 0.2em;
-      overflow-x: hidden;
       word-break: break-word;
       overflow-wrap: break-word;

```

**Comment by Philipinho** [comment]
Fixed in 0.2.10.
